### PR TITLE
React Group Project

### PR DESCRIPTION
**Pull request by Rhaegar121**
In this PR, I have completed the following tasks.

1. Created rockets component and render all rockets from the API
2. Created missions component and render all missions from the API
3. Added action and reducers for reservation and cancel the reservation of rockets
4. Added action and reducers for joining and cancel joining the missions
5. Created my profile components and render a list of all reserved rockets and all joined missions
6. Styled according to the wireframe

### CODE REVIEW

File: src/redux/missions/missionsSlice.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+const url = 'https://api.spacexdata.com/v3/missions';
+
+const initialState = {
+  missions: [],
+};
+
+export const fetchMissions = createAsyncThunk('missions/fetchMissions', async () => {
+  // const response = await axios.get(url);
+  // return response.data;
```
Comment (mckent05) [inline src/redux/missions/missionsSlice.js]: - Please delete all commented code from your code base to help make your code clean and easier to maintain 🙏

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @Rhaegar121 & @Chaseknis  , 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- Your application works as expected, well done 👏 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/missions/missionsSlice.js (hunk 1 above)
Comment (Chaseknis) [reply src/redux/missions/missionsSlice.js]: Thank you @mckent05, We're goin' to fix that

**Review by BertrandConxy — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💪 
- No linter errors
- correct flow is used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Chaseknis** [comment]
thank you @BertrandConxy

**Pull request merged**